guides/Operate-production-services.md

```diff
@@ -36,24 +36,29 @@ Some redirections are done at the DNS level. See the "Web Forwarding" tab of Gan
 - `http(s)://www.openfisca.org` -> `https://openfisca.org`
 - `http(s)://doc.openfisca.fr` -> `https://openfisca.org/doc/`
 - `http(s)://www.openfisca.fr` -> `https://fr.openfisca.org`
+- `http(s)://fr.openfisca.org` -> `https://openfisca.org/fr`
 
 > `http(s)://` means that both `http://` and `https://` URLs are redirected.
 
 ## Services
 
-### Documentation
+### Website
 
-Source code available on <https://github.com/openfisca/openfisca-doc/>
+Source code available on <https://github.com/openfisca/openfisca.org/>
 
-It is deployed to <https://openfisca.org/doc/>.
+It is deployed to <https://openfisca.org/> by continuous deployment provided by CircleCI.
+
+It is hosted by GitHub Pages.
 
-It is hosted by [Netlify](https://www.netlify.com/).
+The [settings](https://github.com/openfisca/openfisca.org/settings/pages) of the GitHub repository defines a custom domain name (`openfisca.org`), and the registrar has been set up to point the `A` and `AAAA` DNS records to GitHub Pages' servers.
 
-The CI on this repository generates the HTML from the documentation and pushes it to the `doc-html` branch through [`.circleci/config.yml`](https://github.com/openfisca/openfisca-doc/blob/master/.circleci/config.yml) and [`publish.sh`](https://github.com/openfisca/openfisca-doc/blob/master/publish.sh).
+### Documentation
+
+Source code available on <https://github.com/openfisca/openfisca-doc/>
 
-Then the CI of [openfisca.org](https://github.com/openfisca/openfisca.org) fetches that branch and includes its contents in the build, then updates the published version on GitHub Pages through [`publish-doc.sh`](https://github.com/openfisca/openfisca.org/blob/master/publish-doc.sh).
+It is deployed to <https://openfisca.org/doc/>.
 
-The [settings](https://github.com/openfisca/openfisca.org/settings/pages) of this GitHub Pages instance defines a custom domain name (`openfisca.org`), and the `/doc` base path targets a directory in the static build.
+The CD on this repository generates the HTML from the documentation and adds the built content to the `doc` directory of the `gh-pages` branch through the `deploy` GitHub Actions workflow.
```

Comment:
The openfisca-doc generated content is added to the **openfisca.org** gh-pages branch, right? Explicit that the branch is on another repository?
```suggestion
The CD on this repository generates the HTML from the documentation and adds the built content to the `doc` directory of the [openfisca.org `gh-pages` branch](https://github.com/openfisca/openfisca.org/tree/gh-pages) through the `deploy` GitHub Actions workflow.
```

Reply:
Yes, and this is super confusing! Thanks for the suggestion.